### DISCUSSION TITLE
import shape file to Grassgis

### DISCUSSION THREAD

**Discussion by Passau113** [OSGeo/grass#2763] [General]
Hello everyone,

i would like to import shape file in GRASSGIS. can someone help me?

thanks in advance

**Comment by neteler** [comment]
Hi @Passau113 ,

you can use the [graphical user interface](https://grass.osgeo.org/grass-stable/manuals/wxGUI.html) or [v.import](https://grass.osgeo.org/grass-stable/manuals/v.import.html) .
See below for some links:

Further info:
- https://grass.osgeo.org/learn/newcomers/
- https://grass.osgeo.org/grass-stable/manuals/helptext.html#importing-data

Overviews:
- https://grass.osgeo.org/grass-stable/manuals/vectorintro.html#vector-data-import-and-export
- https://grasswiki.osgeo.org/wiki/Importing_data (this is a bit dated, though)
